Runtime/GameLiftCoreApi.cs

```diff
@@ -3,51 +3,59 @@
 
 #if !UNITY_SERVER
 
-using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using Amazon.GameLift;
+using AmazonGameLift.Editor;
+using AmazonGameLiftPlugin.Core;
 using AmazonGameLiftPlugin.Core.ApiGatewayManagement;
 using AmazonGameLiftPlugin.Core.ApiGatewayManagement.Models;
 using AmazonGameLiftPlugin.Core.Latency;
 using AmazonGameLiftPlugin.Core.Latency.Models;
 using AmazonGameLiftPlugin.Core.Shared;
 using AmazonGameLiftPlugin.Core.UserIdentityManagement;
 using AmazonGameLiftPlugin.Core.UserIdentityManagement.Models;
+using UnityEngine;
 
 namespace AmazonGameLift.Runtime
 {
     public class GameLiftCoreApi
     {
         private readonly GameLiftConfiguration _configuration;
-        private readonly bool _isLocalMode;
+        private readonly bool _isAnywhereMode;
+        private CoreApi _coreApi;
+        private readonly IGameServerAdapter _gameServerAdapter;
 
-        public GameLiftCoreApi(GameLiftConfiguration configuration)
+        protected GameLiftCoreApi(GameLiftConfiguration configuration)
         {
             _configuration = configuration;
-            _userIdentity = new UserIdentity(new AmazonCognitoIdentityWrapper(configuration.AwsRegion));
-            _apiGateway = new ApiGateway(_userIdentity, new JwtTokenExpirationCheck(), new HttpClientWrapper());
-
-            if (_configuration.ApiGatewayEndpoint.StartsWith("https://localhost"))
+            _coreApi = CoreApi.SharedInstance;
+            var fleetId = _coreApi.GetSetting("FleetId").Value;
+            var fleetLocation = _coreApi.GetSetting("FleetLocation").Value;
+            if (_configuration.IsGameLiftAnywhere)
```

Comment:
Could we add a comment here that `"This property is for local testing. If one wants a production Anywhere fleet, one should not use this property and instead update their deployment scenario"` or something similar?

Basically I'm concerned that we're equating Anywhere with testing locally whereas Anywhere can facilitate local testing but can also be used identically to Managed EC2 in a production environment. This is not something we need to solve right now though, which is why I suggest adding the comment.